packages/main/src/plugin/locked-configuration.spec.ts

```diff
@@ -0,0 +1,177 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import * as fs from 'node:fs';
+
+import { afterEach, describe, expect, test, vi } from 'vitest';
+
+import {
+  SYSTEM_LOCKED_FILE_LINUX,
+  SYSTEM_LOCKED_FILE_MAC,
+  SYSTEM_LOCKED_FILE_WINDOWS_DIR,
+  SYSTEM_LOCKED_FILE_WINDOWS_FILE,
+} from '/@api/configuration/system-defaults.js';
+
+import * as util from '../util.js';
+import { LockedConfiguration } from './locked-configuration.js';
+
+const isMacMock = vi.spyOn(util, 'isMac');
+const isWindowsMock = vi.spyOn(util, 'isWindows');
+const isLinuxMock = vi.spyOn(util, 'isLinux');
```

Comment:
Why are you using spyOn here, and `vi.mock('../util.js',` in other test files ? This is not consistent

Reply:
To change a value, you should use `vi.mocked(isMac).mockReturnValue(true)` and use the appropriate import